preview-env/comment/README.md

```diff
@@ -0,0 +1,155 @@
+    - name: Deploy Preview Environment for ${{ matrix.product_context }}
+        uses: camunda/infra-global-github-actions/preview-env/create@main
+        id: deploy
+        with:
+        revision: ${{ env.BRANCH_NAME }}
+        argocd_token: ${{ steps.secrets.outputs.ARGOCD_TOKEN }}
+        app_name: app-name-${{ matrix.product_context }}
+        app_url: https://${{ matrix.product_context }}.app-name.camunda.cloud
+        argocd_arguments: ${{ env.argocd_arguments }}
+  #########################################################################
+  # Summarize Deployment Results
+  summarize:
+    name: summarize-deployment-results
+    runs-on: ubuntu-22.04
+    if: always() && needs.deploy-preview.result != 'skipped'
+    needs:
+    - deploy-preview
+    steps:
+      - uses: camunda/infra-global-github-actions/preview-env/comment@main
+```
+
+### Delete Summary
+If you want to delete the summary comment you can simply call it (like outlined above) in a different context where no matching workflow artifacts have been uploaded (e.g. your `teardown` workflow).
+Again, 2 examples:
+
+```yaml
+#################
+# Simple Workflow
+
+...
+jobs:
+  teardown:
+    ...
+    - name: Delete Deployment Results
+      if: always() && needs.deploy-preview.result != 'skipped'
+      uses: camunda/infra-global-github-actions/preview-env/comment@main
+    ...
+
+#################
+# Matrix Workflow
+...
+jobs:
+  ...
+  teardown-stuff: ...
+  ...
+  comment:
+    name: delete-deployment-results
+    runs-on: ubuntu-22.04
+    if: always() && needs.teardown.result != 'skipped'
+    needs:
+    - teardown
+    steps:
+      - uses: camunda/infra-global-github-actions/preview-env/comment@gha-459-preview-env-ux
+  ...
+```
+
+### Artifact Production
+The usage scenarios above only apply, when your workflow has produced artifacts beforehands which follow this naming convention:
+```
+# Artifact naming convention
+deployment-status-${{ github.run_id }}-${{ github.run_attempt }}-*
+```
+
+The [`create` action](../create/action.yml) got that part built-in like so:
+```yaml
+- name: Create deployment status artifact
+    if: always()
+    shell: bash
+    run: |
+      if [[ "${{ github.action_status }}" == "success" ]]; then
+        echo ":full_moon_with_face: [\`${{ inputs.app_name }}\`](${{ inputs.app_url }}): ${{ github.action_status }}\n" > status_${{ inputs.app_name }}.md
+      else
+        echo ":boom: \`${{ inputs.app_name }}\`: ${{ github.action_status }} :arrow_right: see [ArgoCD](https://${{ inputs.argocd_server }}/applications?health=Progressing%2CDegraded&search=${{ inputs.app_name }})\n" > status_${{ inputs.app_name }}.md
+      fi
+  - name: Upload deployment status as artifact
+    if: always()
+    uses: actions/upload-artifact@v4
+    with:
+      name: deployment-status-${{ github.run_id }}-${{ github.run_attempt }}-${{ inputs.app_name }}
+      path: status_${{ inputs.app_name }}.md
+      retention-days: 1
+```
```

Comment:
This part is not up to date, due to the recent changes. I would replace it by a single comment and/or link.